### PR TITLE
A mass change from http://xbmc.org to http://kodi.tv.

### PR DESCRIPTION
**Pull request by kekePower**
Change http://xbmc.org to http://kodi.tv

## Description
I used the command
grep --include={\*.cpp,\*.h} -rnl './' -e "http://xbmc.org" | xargs -i@ sed -i 's/http\:\/\/xbmc\.org/http\:\/\/kodi\.tv/g' @
to change every http://xbmc.org to http://kodi.tv in *.cpp and *.h files

## Motivation and Context
I read in a PR once that it was a wish to switch all references to xbmc.org over to kodi.tv.
When I found a great grep/sed command that did the trick in one go, I decided I wanted to give it a try.

A lot of files has been touched, so if you want to reject this PR, I'm fine with it.

## How Has This Been Tested?
No need to test as this only touches references to http://xbmc.org.

## Types of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: tools/depends/native/JsonSchemaBuilder/src/JsonSchemaBuilder.cpp [hunk 1]
```diff
@@ -153,7 +153,7 @@ int main(int argc, char* argv[])
 
   out << "namespace JSONRPC" << endl;
   out << "{" << endl;
-  out << "  const char* const JSONRPC_SERVICE_ID          = \"http://xbmc.org/jsonrpc/ServiceDescription.json\";" << endl;
+  out << "  const char* const JSONRPC_SERVICE_ID          = \"http://kodi.tv/jsonrpc/ServiceDescription.json\";" << endl;
```
Comment (notspiff) [inline tools/depends/native/JsonSchemaBuilder/src/JsonSchemaBuilder.cpp]: remove this change please. affects RPC, needs to verify it won't break things and don't want to apply brain in this pr. you can open a separate pr for it if you cba

File: xbmc/platform/Filesystem.h [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #pragma once
 /*
 *      Copyright (C) 2005-2017 Team Kodi
-*      http://xbmc.org
+*      http://kodi.tv
```
Comment (notspiff) [inline xbmc/platform/Filesystem.h]: a windows eol snuck in.

File: xbmc/platform/win32/Filesystem.cpp [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /*
  *      Copyright (C) 2005-2017 Team Kodi
- *      http://xbmc.org
+ *      http://kodi.tv
```
Comment (notspiff) [inline xbmc/platform/win32/Filesystem.cpp]: windows eol (nope, not okay in windows files either).

File: xbmc/utils/Literals.h [hunk 4]
```diff
@@ -1,7 +1,7 @@
 #pragma once
 /*
  *      Copyright (C) 2014 Team XBMC
- *      http://xbmc.org
+ *      http://kodi.tv
```
Comment (notspiff) [inline xbmc/utils/Literals.h]: windows eol

**Review by notspiff — CHANGES_REQUESTED** [review]
RPC change is hard, the others i realize are not your doing but if cba to address them while at it, you get an extra cookie.

**Comment by kekePower** [comment]
@notspiff 
I do like cookies :-)

**Comment by notspiff** [comment]
now to get sugar coating; you should put the json part of the second commit into the first. you know your way around git squashery ?

**Comment by kekePower** [comment]
@notspiff 
Do you want only the json part or is it OK to sqash the whole second commit?

**Comment by Rechi** [comment]
wait for #13427 getting merged and then rebase onto master again (and squash to a single commit)

**Comment by kekePower** [comment]
@Rechi Ok, will wait.

**Comment by kekePower** [comment]
@notspiff 
No, I guess I really don't know anything about git squash even though I tried to read up on it.

**Comment by notspiff** [comment]
okay, then i will handle that part. no worries.

**Comment by kekePower** [comment]
@notspiff Tusen takk :-)

**Comment by notspiff** [comment]
bare hyggelig.

pushed manually in rebased  form. thanks for your contribution!

**Comment by da-anda** [comment]
while at it, I guess we should also replace `Team XBMC` with `Team Kodi` :)

**Pull request closed without merging**